components/nodemanager-service/managers/managers_test.go

```diff
@@ -39,3 +39,19 @@ func TestGetAWSCredsSetsRegionWhenProvided(t *testing.T) {
 		Region:          "user-provided-region",
 	}, creds)
 }
+//AZURE
+func TestAzureCredsWithOutSubscriptionID(t *testing.T) {
```

Comment:
can we add another test with subscription id?

Reply:
we can but it is no use to add instead of an empty string, we check with a string value
This will be``assert.Equal(t, "", subscriptionID)`` -> ```assert.Equal(t, "fake-3", subscriptionID)```

Reply:
Sure that's correct but if you add it to []*query.Kv{}, we will be able to know if GetAzureCreds processes them properly.

Reply:
cool, I will add

Reply:
Thanks @shaik80 , its done